plugins/mcreator-localization/lang/texts.properties

```
@@ -1391,8 +1392,7 @@ blockly.block.aitasks_container=AI task / goals list
 blockly.block.aitasks_container.tip=Tasks higher on the list have a higher priority
 blockly.block.aitasks_container.tooltip=This is the starting point of the AI task list.
 blockly.block.cmdargs_start=Command entry point
-blockly.block.advancement_trigger=Grant this advancement to the player
-blockly.block.advancement_trigger.tooltip=Connect one trigger block that triggers this advancement to this block
+blockly.block.advancement_trigger=Grant if all criteria below are met
```

```suggestion
blockly.block.advancement_trigger=Grant if ALL criteria below are met
```
I think that highlighting ALL could be useful to make others like me understand that this doesn't work like the procedures (if they don't read the initial part where the parameters are added to have the advancement)

---

I don't think this would help much to attract the attention.

---

Ok